translations/tiled_ru.ts

```diff
@@ -4733,8 +4733,8 @@ Please select specific format.</source>
     <message numerus="yes">
         <location filename="../src/tiled/flipmapobjects.cpp" line="+39"/>
         <source>Flip %n Object(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>Повернуть %n объект(ов)</numerusform>
```

Comment:
Don't sure if I understand correctly but I try to reply. In this case it could be 2 ways of Russian word endings depending of actual amount of objects. Example:
- 4 объект(а)
- 5 объект(ов)

In such case what type of fix you expect from me?
- Merge both endings: Повернуть %n объект(ов/а)
- Choose one default: Повернуть %n объектов
- Use no ending: Повернуть %n объект (this sounds strange because such form is only correct for singular forms.